source/PluginDev/Assets/GooglePlayGames/Editor/template-AndroidManifest.txt

```diff
@@ -8,6 +8,9 @@
     
     <uses-sdk android:minSdkVersion="8" android:targetSdkVersion="16" /> 
  
+    <uses-permission android:name="android.permission.GET_ACCOUNTS"/>
```

Comment:
Yeah, this was one of those things that I couldn't see exactly how to get the template to have optional things like this in.  If possible, do you think you could work the optionality in?